app/src/main/java/com/lab/skuld/ui/App.kt

```diff
@@ -39,16 +37,56 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.google.firebase.Timestamp
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.ktx.auth
+import com.google.firebase.firestore.DocumentId
 import com.google.firebase.ktx.Firebase
-import com.lab.skuld.ui.screens.LoadingState
 import com.lab.skuld.ui.screens.ShowCalendarScreen
 import com.lab.skuld.ui.screens.ShowLoginScreen
 import com.lab.skuld.ui.screens.ShowNewNoteScreen
+import com.lab.skuld.ui.screens.ShowNoteScreen
 import com.lab.skuld.ui.screens.ShowTasksScreen
 import com.lab.skuld.ui.theme.SkuldFrontendTheme
 import kotlinx.coroutines.launch
+import java.util.Date
+
+class MaybeEvent {
+    @DocumentId
+    val id = ""
+    val startDate: Timestamp? = null
+    val endDate: Timestamp? = null
+    val title: String? = null
+    val checked: Boolean? = null
+    val contents: String? = null
+}
+
+data class Event(
+    val id: String,
+    val startDate: Date?,
+    val endDate: Date?,
+    val title: String,
+    val checked: Boolean?,
+    val contents: String?,
+)
+
+fun maybeToEvent(maybe: MaybeEvent): Event? {
+    if ((maybe.startDate == null && maybe.endDate == null) || maybe.title == null)
+        return null
+
+    return Event(
+        maybe.id,
+        maybe.startDate?.toDate(),
+        maybe.endDate?.toDate(),
+        maybe.title,
+        maybe.checked,
+        maybe.contents
+    )
+}
```

Comment:
Please don't clutter the main compose file, if you need a class from another file, import it, or move it into a standalone file in `model` package